README.md

```diff
@@ -59,18 +59,24 @@ cdm2bxc.sh -h
 ```
 # Initialize a new migration project (named gilmer_demo, from CDM collection gilmer)
 cdm2bxc.sh init -p gilmer_demo -c gilmer
+
 cd gilmer
```

Comment:
I think this should be `cd gilmer_demo`.

Reply:
Thanks, updated